data/brands/shop/convenience.json

```diff
@@ -3170,6 +3192,28 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Migros",
+      "id": "migros-986a24",
+      "locationSet": {"include": ["001"]},
+      "tags": {
+        "brand": "Migros",
+        "brand:wikidata": "Q680727",
```

Comment:
`brand:wikidata` should also refer to `Q1754510`

Reply:
Changed wikidata reference